lib/TWCManager/Vehicle/TeslaAPI.py

```diff
@@ -1435,6 +1441,7 @@ def get_car_api(self, url, checkReady=True, provesOnline=True):
                 elif req.status_code == 429:
                     # We're explicitly being told to back off
                     self.errorCount = max(30, self.errorCount)
+                    self.updateCarApiLastErrorTime()
```

Comment:
I think this needs to be:

` self.carapi.updateCarApiLastErrorTime(self)`

We are in the `CarApiVehicle` class and `updateCarApiLastErrorTime` is defined in the `TeslaAPI` class.

Reply:
Thanks, fixed.